package.json

```diff
@@ -83,7 +83,7 @@
     "cucumber-html-reporter": "5.5.1",
     "depcheck": "1.4.3",
     "ejs": "3.1.9",
-    "eslint": "8.40.0",
+    "eslint": "8.43.0",
```

Comment:
I bumped eslint as locally calling eslint --fix tried always to alter packages/web-pkg/src/composables/router/types.ts
which resulted in a linter error in the ci, which was annoying